microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -995,12 +1058,30 @@ Status:
 _new as of Microsoft Edge 118_
 
 Whether to enable the Storage Buckets Tree in the **Application** tool.
-<!-- todo: need text -->
+<!-- todo old: need text -->
 
 <!-- 
 See also:
 * []() -->
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable context menu that allows to modify trees in the Flame Chart
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
+
+Enables the right-click menuitem that allows modifying trees in the flame chart in the **Performance** tool.
+<!-- todo: details, png -->
```

Comment:
4. @captainbrosset @zoherghadyali - need info about the Experiments checkbox:
Enable context menu that allows to modify trees in the Flame Chart

Reply:
I didn't manage to see this in Edge, but it's in Chrome Canary already. Maybe we just haven't gotten it in Edge yet and will in a few days.
Here's what this experiment does:

![image](https://github.com/MicrosoftDocs/edge-developer/assets/1152698/f906f256-2287-4158-ad44-d27107d8ff8f)

With the experiment enabled, you can right-click events in the flame chart of the Performance tool, and modify the chart at that location.